config/customFormats.js

```diff
@@ -13,6 +15,51 @@ const getLeaf = (object) => {
   };
 };
 
+function MDX(dictionary) {
+  const topLevels = Object.entries(dictionary.properties);
+  let str = ``;
+
+  function MDXContent(dictionary, level, first) {
+    const category = dictionary[0];
+
+    let isToken = dictionary[1].value;
+
+    if (isToken) {
+      str += `\n| ${dictionary[0]} | ${dictionary[1].value} |`;
+    } else {
+      const subdictionary = Object.entries(dictionary[1]);
+      const hasProperties = subdictionary.every(function(item) {
+        return item[1].value;
```

Comment:
We'll probably want to do the same thing here 
```suggestion
        return item[1].hasOwnProperty("value");
```